### PR TITLE
[pt-BR] Updates basic enum to latest version

### PR DESCRIPTION
**Pull request by pragmaticivan**

### CODE REVIEW

File: pt/lessons/basics/enum.md [hunk 1]
```diff
@@ -9,13 +9,33 @@ Um conjunto de algoritmos para fazer enumeração em coleções.
...
+at/2
+at/3
+...
+```
+
+Usando isso, é claro que temos uma grande quantidade de funcionalidades, e isso é por uma razão.
+A enumeração é o núcleo da programação funcional e é uma coisa incrivelmente útil.
+Ao aproveitar isso combinado com outras vantagens de Elixir, como a documentação sendo um cidadão de primeira classe como acabamos de ver, também pode ser incrivelmente poderoso para o desenvolvedor.
+
+Para obter uma lista completa de funções, visite a documentação oficial do [`Enum`](https://hexdocs.pm/elixir/Enum.html); Para enumeração preguiçosada use o módulo [`Stream`](https://hexdocs.pm/elixir/Stream.html).
```
Comment (anderkonzen) [inline pt/lessons/basics/enum.md]: I believe the word "Para" could be changed to lowercase in this case - after the semicolon. What do you think?
Also, there is a small typo in "preguiçosada" -> "preguiçosa".

File: pt/lessons/basics/enum.md [hunk 2]
```diff
@@ -119,30 +140,35 @@ iex> Enum.max([], fn -> :bar end)
...
+
+`sort/1` usa a funcionalidade de ordenação do Erlang para determinar a ordem:
 
 ```elixir
 iex> Enum.sort([5, 6, 1, 3, -1, 4])
 [-1, 1, 3, 4, 5, 6]
 
 iex> Enum.sort([:foo, "bar", Enum, -1, 4])
 [-1, 4, Enum, :foo, "bar"]
+```, Enum, :foo, "bar"]
```
Comment (anderkonzen) [inline pt/lessons/basics/enum.md]: Comparing to the English version, I believe this line was added unintentionally.

**Comment by pragmaticivan** [comment]
Thanks @anderkonzen ! Just sent the changes!

**Comment by anderkonzen** [comment]
Super!

**Pull request merged**